src/traits.rs

```diff
@@ -121,6 +121,18 @@ pub trait RootContext: Context {
     fn on_queue_ready(&mut self, _queue_id: u32) {}
 
     fn on_log(&mut self) {}
+
+    fn create_http_context(&self, _context_id: u32) -> Box<dyn HttpContext> {
+        Box::new(EmptyHttpContext)
```

Comment:
`Box<dyn HttpContext>` -> `Option<Box<dyn HttpContext>>` and return `None`.

Reply:
done